Reason: (e) sulu/skeleton 3.0 links the unversioned latest Sulu docs on image formats.
Evidence: https://docs.sulu.io/3.x/book/image-formats.html answers 200 without a redirect (curl, 2026-10-04).

--- a/config/image-formats.xml
+++ b/config/image-formats.xml
@@ -3,7 +3,7 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://schemas.sulu.io/media/formats http://schemas.sulu.io/media/formats-1.1.xsd">
 
-    <!-- See: http://docs.sulu.io/en/latest/book/image-formats.html how to define image-formats -->
+    <!-- See: https://docs.sulu.io/3.x/book/image-formats.html how to define image-formats -->
 
     <!--
     <format key="300x">
